doc/api/vm.md

```diff
@@ -570,7 +570,7 @@ changes:
   * `displayErrors` {boolean} When `true`, if an [`Error`][] error occurs
     while compiling the `code`, the line of code causing the error is attached
     to the stack trace.
-  * `timeout` {number} Specifies the number of milliseconds to execute `code`
+  * `timeout` {integer} Specifies the number of milliseconds to execute `code`
```

Comment:
Is the `This value must be a strictly positive integer.` note missing here?

Reply:
Fixed

Reply:
Fixed.